### PR TITLE
Milestone 2 : Components

### PR DESCRIPTION
**Pull request by aboongm**
- Deleted all the boilerplate from CRA (text, images, styles).
- Created a directory called components.
- Inside components, created a new Calculator.js file.
- In Calculator.js, created a React component that matches original design given
- Only the UI of the calculator is developed
- Used class based components
- Calculator is imported and used in the main component (App.js).
- Tested it in the browser; it renders without problems.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
-# math-magician
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# React Project: Math Magician
```
Comment (TedLivist) [inline README.md:1]: - [ ] You are yet to delete all unnecessary boilerplates files. Kindly delete the `public/robots.txt` and `src/setupTests.js` files. This way we rid our project of unnecessary files and keep it clean.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @aboongm,

Good job so far!
✔️ Good documentation
✔️ Class-based components
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (aboongm) [reply README.md:1]: @TedLivist The README file is part of Microverse Template. Should I remove that too?

**Review by mikethreels — APPROVED** [review]
Hi Aboong, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**